httpobs/website/api.py

```diff
@@ -24,15 +24,28 @@ def api_post_scan_hostname():
     hostname = request.args.get('host', '').lower()
 
     # Fail if it's not a valid hostname (not in DNS, not a real hostname, etc.)
+    ip = True if valid_hostname(hostname) is None else False
     hostname = valid_hostname(hostname) or valid_hostname('www.' + hostname)  # prepend www. if necessary
-    if not hostname:
-        return {'error': '{hostname} is an invalid hostname'.format(hostname=request.args.get('host', ''))}
+
+    if ip:
+        return {
+            'error': 'invalid-hostname-ip',
+            'text': '{hostname} is an IP address, not hostname'.format(hostname=request.args.get('host', '')),
```

Comment:
I'm not sure I understand, but it's not the best worded. I'll try to make it more readable.